modules/oadp-about-kopia-repo-maintenance.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-deleting-backups-doc.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-about-kopia-repo-maintenance_{context}"]
+= About Kopia repository maintenance
+
+Following are the two types of Kopia repository maintenance:
+
+* Quick maintenance:
+
+** Runs every hour to keep the number of index blobs (n) low. A high number of indexes negatively affects the performance of Kopia operations.
+** Does not delete any metadata from the repository without ensuring that another copy of the same metadata exists.
+
+
+* Full maintenance:
+
+** Runs every 24 hours to perform garbage collection of repository contents that are no longer needed.
+** `snapshot-gc`, a full maintenance task, finds all files and directory listings that are no longer accessible from snapshot manifests and marks them as deleted.
+** A full maintenance is a resource-costly operation, as it requires scanning all directories in all snapshots that are active in the cluster. 
+
+== Kopia maintenance in {oadp-short}
+
+* The `repo-maintain-job` jobs are executed in the namespace where {oadp-short} is installed. Following is an example of the jobs:
++
+[source, terminal]
+----
+pod/repo-maintain-job-173...2527-2nbls                             0/1     Completed   0          168m
+pod/repo-maintain-job-173....536-fl9tm                             0/1     Completed   0          108m
+pod/repo-maintain-job-173...2545-55ggx                             0/1     Completed   0          48m
+----
+
+* You can check the logs of the `repo-maintain-job` for more details about the cleanup and the removal of artifacts in {aws-short} S3 storage. You can find a note, as shown in the following example, in the `repo-maintain-job` when the next full cycle maintenance is due.
```

Comment:
Issue not specific to AWS S3 bucket. I think we should make the statement generic for all the buckets.
```suggestion 
* You can check the logs of the `repo-maintain-job` for more details about the cleanup and the removal of artifacts in backup storage. You can find a note, as shown in the following example, in the `repo-maintain-job` when the next full cycle maintenance is due.
```